src/App.js

```diff
@@ -24,6 +27,9 @@ function App() {
           <Route path='/admin-app/students/:id' exact component={StudentDetailPage} />
           <Route path={['/admin-app/students', '/']} exact component={StudentsPage} />
           <Route path='/admin-app/payments' exact component={PaymentsPage} />
+          <Route path='/admin-app/teachers' exact component={TeachersPage} />
+          <Route path='/admin-app/teachers/:id/edit' exact component={EditTeacherForm} />
+          <Route path='/admin-app/teachers/add' exact component={CreateTeacherForm} />
           <Route path='/admin-app/payments/:id' exact component={PaymentsByStudent} />
```

Comment:
PaymentsByStudent улетел вниз
```suggestion
          <Route path='/admin-app/payments/:id' exact component={PaymentsByStudent} />
          <Route path='/admin-app/teachers' exact component={TeachersPage} />
          <Route path='/admin-app/teachers/:id/edit' exact component={EditTeacherForm} />
          <Route path='/admin-app/teachers/add' exact component={CreateTeacherForm} />
```

Reply:
исправлен